### PR TITLE
Added gamepad profiles for STK-7039RG

### PR DESCRIPTION
**Pull request by cjdg**
Gamepad profiles related with the STK-7039RG

### CODE REVIEW

**Comment by cjdg** [comment]
I added a small documentation for common profiles for the Saitek STK-7039RGB controller

File: udev/7039RG.cfg [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+input_l3_btn = "13"
+input_r3_btn = "14"
+input_l_x_plus_axis = "+0"
+input_l_x_minus_axis = "-0"
+input_l_y_plus_axis = "+1"
+input_l_y_minus_axis = "-1"
+input_r_x_plus_axis = "+2"
+input_r_x_minus_axis = "-2"
+input_r_y_plus_axis = "+3"
+input_r_y_minus_axis = "-3"
```
Comment (RobLoach) [inline udev/7039RG.cfg:82]: Would you mind added the button labels here too?

File: udev/Afterglow Prismatic Wired Controller.cfg [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+input_l3_btn = "9"
+input_r3_btn = "10"
+input_l_x_plus_axis = "+0"
+input_l_x_minus_axis = "-0"
+input_l_y_plus_axis = "+1"
+input_l_y_minus_axis = "-1"
+input_r_x_plus_axis = "+3"
+input_r_x_minus_axis = "-3"
+input_r_y_plus_axis = "+4"
+input_r_y_minus_axis = "-4"
```
Comment (RobLoach) [inline udev/Afterglow Prismatic Wired Controller.cfg:28]: Button labels :pray:

File: udev/SHANWAN Android Gamepad.cfg [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+input_r3_btn = "14"
+input_l_x_plus_axis = "+0"
+input_l_x_minus_axis = "-0"
+input_l_y_plus_axis = "+1"
+input_l_y_minus_axis = "-1"
+input_r_x_plus_axis = "+2"
+input_r_x_minus_btn = "0"
+input_r_y_plus_axis = "+3"
+input_r_y_minus_axis = "-3"
+input_r_x_minus_axis = "-2"
```
Comment (RobLoach) [inline udev/SHANWAN Android Gamepad.cfg:53]: Button labels would help here :pray:

File: udev/ZhiXu Gamepad.cfg [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+input_l3_btn = "13"
+input_r3_btn = "14"
+input_l_x_plus_axis = "+0"
+input_l_x_minus_axis = "-0"
+input_l_y_plus_axis = "+1"
+input_l_y_minus_axis = "-1"
+input_r_x_plus_axis = "+2"
+input_r_x_minus_axis = "-2"
+input_r_y_plus_axis = "+3"
+input_r_y_minus_axis = "-3"
```
Comment (RobLoach) [inline udev/ZhiXu Gamepad.cfg:81]: Mind some button labels?

**Review by RobLoach — CHANGES_REQUESTED** [review]
Thank you so much. While we would usually prefer to handle the permissions in a different Pull Request, I think it's fine to have them here. Thanks for the clean up.

Mind adding some of the button labels? Then they'll appear cleanly in the UI.

**Comment by RobLoach** [comment]
Actually, going to merge this for now. Would you mind adding the button labels in?

**Pull request merged**